### PR TITLE
Included the basic setup for the search feature

### PR DESCRIPTION
**Pull request by ez-may**
Included the JSP files required to begin work on the nav bar and basic structure to begin templating.

### CODE REVIEW

File: src/main/webapp/pages/mainPage.jsp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Home</title>
+  </head>
+  <body>
+      <%@ include file="topbar.jsp"%>
+      <p> testing the defaut page</p>
```
Comment (isamitre) [inline src/main/webapp/pages/mainPage.jsp]: nit: default is spelled wrong

**Review by isamitre — APPROVED** [review]

File: src/main/webapp/pages/mainPage.jsp (hunk 1 above)
Comment (ez-may) [reply src/main/webapp/pages/mainPage.jsp]: Ack and fixed

File: src/main/webapp/components/topbar.jsp [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div id="top-bar">
```
Comment (AlexanderEllis) [inline src/main/webapp/components/topbar.jsp:3]: Do you want to add a comment (either here or in the readme) on how to use this for future pages?

File: src/main/webapp/css/style.css [hunk 3]
```diff
@@ -1,15 +1,9 @@
-body {
-  margin: 0;
-  padding: 0;
+#content {
+  margin-left: auto;
```
Comment (AlexanderEllis) [inline src/main/webapp/css/style.css:2]: Does `margin: auto` work here instead of these two lines?

**Review by AlexanderEllis — APPROVED** [review]
LGTM! Do you have a screenshot of this in action? Might be helpful for the PR description for the future.

File: src/main/webapp/css/style.css (hunk 3 above)
Comment (ez-may) [reply src/main/webapp/css/style.css:2]: Not entirely sure, I pulled this from the master branch.

File: src/main/webapp/components/topbar.jsp (hunk 2 above)
Comment (ez-may) [reply src/main/webapp/components/topbar.jsp:3]: Ack and will include

**Comment by ez-may** [comment]
image
This is what the basic implementation currently looks like

**Pull request merged**